opts/ip.go

```diff
@@ -5,20 +5,25 @@ import (
 	"net"
 )
 
-// IpOpt type that hold an IP
-type IpOpt struct {
+// IPOpt holds an IP
```

Comment:
But why does it hold an IP? What is it for?

I think generally for all these types we have we can probably just add that these are used for storing values from CLI flags.